src/zenml/zen_stores/schemas/step_run_schemas.py

```diff
@@ -348,7 +348,7 @@ class StepRunOutputArtifactSchema(SQLModel, table=True):
 
     # Fields
     name: str
-    type: StepRunOutputArtifactType
```

Comment:
Just commenting as a reminder for myself to check that this is handled correctly in wherever this is used.